odbc/src/api/functions.rs

```diff
@@ -1776,58 +1782,41 @@ unsafe fn sql_get_env_attrw_helper(
     SqlReturn::SUCCESS
 }
 
-///
-/// [`SQLGetInfoW`]: https://learn.microsoft.com/en-us/sql/odbc/reference/syntax/SQLGetInfo-function
-///
-/// This is the WideChar version of the SQLGetInfo function
-///
-/// # Safety
-/// Because this is a C-interface, this is necessarily unsafe
-///
-#[named]
-#[no_mangle]
-pub unsafe extern "C" fn SQLGetInfoW(
-    connection_handle: HDbc,
-    info_type: USmallInt,
-    info_value_ptr: Pointer,
-    buffer_length: SmallInt,
-    string_length_ptr: *mut SmallInt,
-) -> SqlReturn {
-    panic_safe_exec_clear_diagnostics!(
-        debug,
-        || sql_get_infow_helper(
-            connection_handle,
-            info_type,
-            info_value_ptr,
-            buffer_length,
-            string_length_ptr
-        ),
-        connection_handle
-    )
-}
+macro_rules! sql_get_info_helper {
+($connection_handle:ident,
+ $info_type:ident,
+ $info_value_ptr:ident,
+ $buffer_length:ident,
+ $string_length_ptr:ident,
+ $string_func:path,
```

Comment:
apologies if I'm missing it, but curious why this became a param, it seems like both calls to this macro use the same value for this

Reply:
One call passes "SQLInfo" the other passes "SQLInfoW", it's only used in tracing :) Though thanks for bringing this up, because it should also be used in the error message.

Reply:
Oh you mean the string function. This was meant to be an ansi version of the string function, but this is actually working for power bi. That's troubling.